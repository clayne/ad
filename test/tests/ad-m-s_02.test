ad | -c always -m -s0 | row_separator.txt | 0
