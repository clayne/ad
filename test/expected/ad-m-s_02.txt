[32m[K0000000000000000[m[K[36m[K:[m[K [41;1m[K30[m[K31 3233 3435 3637 3839 4142 4344 450A  [41;1m[K0[m[K123456789ABCDE.
[36m[K----------------
[m[K[32m[K0000000000000020[m[K[36m[K:[m[K [41;1m[K30[m[K31 3233 3435 3637 3839 4142 4344 450A  [41;1m[K0[m[K123456789ABCDE.
